Team#31 Theme#2

anuj-2001 opened the pull request.

anuj-2001 committed:
Create Hello

anuj-2001 committed:
Delete Hello

anuj-2001 committed:
Created Readme.md

anuj-2001 committed:
Update Readme.md

raunakjaiswal committed:
Add files via upload

raunakjaiswal committed:
Delete main.dart

raunakjaiswal committed:
Add files via upload

theyashshahs commented:
@anuj-2001 change the PR title to include the theme number as well and please remove the "first PR" from it.

anuj-2001 committed:
Working on database

raunakjaiswal committed:
Email sending code uploaded

anuj-2001 committed:
Working on Registration Form

raunakjaiswal committed:
we has jst made registered  form linked with firebase

raunakjaiswal committed:
Delete aftermain.dart

raunakjaiswal committed:
Delete main.dart

raunakjaiswal committed:
Delete needregister.dart

raunakjaiswal committed:
we have just made registration details linked with firebase

anuj-2001 committed:
Separate classes for User registration

anuj-2001 committed:
Pin code based search optimization

anuj-2001 committed:
Library files uploaded

anuj-2001 committed:
Apk of the application

anuj-2001 committed:
Create Readme.md

anuj-2001 committed:
Add files via upload

anuj-2001 committed:
Add files via upload

anuj-2001 committed:
Add files via upload

anuj-2001 committed:
Create Readme.md

anuj-2001 committed:
Final Commit

All the generated code also pushed for reference and drive link for the project also added. APK file for the application is also uploaded in the folder.
